src/Features/Core/Portable/GenerateMember/GenerateConstructor/GenerateConstructorHelpers.cs

```diff
@@ -80,7 +80,11 @@ private static bool IsSymbolAccessible(Compilation compilation, ISymbol symbol)
             for (var i = 0; i < constructor.Parameters.Length; i++)
             {
                 var constructorParameter = constructor.Parameters[i];
-                var conversion = semanticFacts.ClassifyConversion(semanticModel, expressions[i], constructorParameter.Type);
+                var constructorParameterType = SymbolKey.Create(constructorParameter.Type).Resolve(semanticModel.Compilation).Symbol as ITypeSymbol;
```

Comment:
doc why.

Reply:
alternatively, we can just skip some of this stuff if the symbo's language and the compilation's language are not hte same.  i'd honestly probably prefer that.  but i can go with this approach.

also, instead of resolivng each type, just try to resolve the construct itself into the compilation.

Reply:
Yeah didn't want to entirely break the feature so kept it as is. Will fix resolving constructor itself.